### ISSUE TITLE
A11y Code Sprint Berlin: Success Criterion 9.2.4.1 - wrap page content in <main>

### ISSUE THREAD

**Issue by SarahEdding** [benjaminkott/bootstrap_package#1293]
# Feature Request

## Description
Screen reader users navigate pages by way of certain semantic elements (landmarks), `nav` `header` and `main `among them. All relevant content of any given page should be in a `main` container. 

## Solution
Make sure all content pages have a `main` and that relevant contend is inside it. This is currently not the case for the [home page](https://bitv-v12.live.typo3.cps.321.works/).

## Alternative
For the home page, provide sections or articles, each with their own header, for individual blocks of content. Note: this requires a certain level of skill in editors to use.

**Comment by jwundermann** [comment]
@alexakressmann please assign to me
